.gitlab-ci.yml

```diff
@@ -560,6 +561,38 @@ build/amd/openmpi/clang/hip/release/static:
     BUILD_TYPE: "Release"
     BUILD_SHARED_LIBS: "OFF"
 
+build/amd/openmpi/gcc/hip502/debug/shared:
+  <<: *default_build_with_test
+  extends:
+    - .quick_test_condition
+  tags:
+    - private_ci
+    - amdci
+    - gpu
+  image: localhost:5000/hpccm_hip_d93a513e_gko-hip502-openmpi-gnu8-llvm11
```

Comment:
Isn't 5.0 already tested with the other HIP job (which would be bleeding edge?). Also, maybe add the versioning to line 547. If that's the case, maybe testing 4.5 here instead makes more sense than having two 5.x?